CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# Changelog
+
+## [v0.3.3](https://github.com/ros-infrastructure/superflore/tree/v0.3.3) (2021-08-07)
+
+[Full Changelog](https://github.com/ros-infrastructure/superflore/compare/v0.3.2...v0.3.3)
+
+**Closed issues:**
+
+- License parsing doesn't parse e.g. LGPLv3 correctly [\#271](https://github.com/ros-infrastructure/superflore/issues/271)
+
+**Merged pull requests:**
+
+- Fix parsing of \<exports\>\<build\_type\> with conditionals [\#281](https://github.com/ros-infrastructure/superflore/pull/281) ([shr-project](https://github.com/shr-project))
```

Comment:
Should there be an entry for https://github.com/ros-infrastructure/superflore/pull/288 ?

Reply:
Yep. Sorry.